src/acceptance-test/java/de/zalando/aruha/nakadi/webservice/BaseAT.java

```diff
@@ -11,11 +22,41 @@
     protected static final String zookeeperUrl = "localhost:2181";
     protected static final String kafkaUrl = "localhost:9092";
 
+    private static final String postgresqlUrl = "jdbc:postgresql://localhost:5432/local_nakadi_db";
+    private static final String username = "nakadi_app";
+    private static final String password = "nakadi";
+
+    protected static final String EVENT_TYPE_NAME = "test-event-type-name";
     protected static final String TEST_TOPIC = "test-topic";
     protected static final int PARTITIONS_NUM = 8;
 
+    private static boolean initialize = true;
+
+    protected JdbcTemplate template;
+    protected ObjectMapper mapper;
+
     static {
         RestAssured.port = PORT;
         RestAssured.defaultParser = Parser.JSON;
     }
+
+    @Before
+    public void setUp() throws Exception {
```

Comment:
I'm not proud of this `setUp`. The fact is that tests used to rely on a topic called "test-topic" which is created by a bash script that initialise local storages. E.g. the tests rely on some external setup.

After the decoupling, topic name is retrieved indirectly through event type table. So the way I found to solve this was to inject in the database the proper entry, since these are acceptance tests and mocking is not an option.